tests/SCF/test_SCF.py

```diff
@@ -1,14 +1,21 @@
 import pytest
 import SCF
+import main
+import pickle
 
+# Overlap Integrals
+Suv = pickle.load(open("suv.pkl", "rb"))

```

Comment:
These are already included in the fixture, with functions that are available to get these matrices, this should be all doen through the fixture as now you have two sets of functionality in here.